evaluate.py

```diff
@@ -176,6 +188,21 @@ def main(args: argparse.Namespace) -> None:
             "learning_rate": model.hparams["learning_rate"],
             "loss": model.hparams["loss"],
         }
+    elif issubclass(TASK, ObjectDetectionTask):
+        val_row = {
+            "split": "val",
+            "detection_model": model.hparams["detection_model"],
+            "backbone": model.hparams["backbone"],
+            "learning_rate": model.hparams["learning_rate"],
+        }
+
+        test_row = {
+            "split": "test",
+            "detection_model": model.hparams["detection_model"],
+            "backbone": model.hparams["backbone"],
+            "learning_rate": model.hparams["learning_rate"],
+        }
+
```

Comment:
Can you get rid of this empty line so it's clear that these are all clauses of the same if-statement?